report/latest_map.go

```diff
@@ -193,15 +193,57 @@ func (m *LatestMap) CodecEncodeSelf(encoder *codec.Encoder) {
 	} else {
 		encoder.Encode(nil)
 	}
+
+	//_, r := codec.GenHelperEncoder(encoder)
+	//if m.Map == nil {
+	//	r.EncodeNil()
+	//	return
+	//}
+	//
+	//r.EncodeMapStart(m.Map.Size())
+	//m.Map.ForEach(func(key string, value interface{}) {
+	//	r.EncodeString(1, key)
+	//	encoder.Encode(value.(LatestEntry))
+	//})
 }
 
+const (
+	containerMapKey   = 2
+	containerMapValue = 3
+	containerMapEnd   = 4
```

Comment:
Unfortunately we're poking around with the internals of our codec library here.  There are constants they chose not to expose.  Want me to stick a comment with a link to the source?